src/core/src/main/java/org/apache/jmeter/save/CSVSaveService.java

```diff
@@ -1104,14 +1104,14 @@ private static boolean isDelimOrEOL(char delim, int ch) {
     public static void saveSampleResult(SampleEvent event, PrintWriter out) {
         SampleSaveConfiguration saveConfiguration = event.getResult().getSaveConfig();
         String delimiter = saveConfiguration.getDelimiter();
-        String savee = resultToDelimitedString(event, event.getResult(), saveConfiguration, delimiter);
-        out.println(savee);
+        String save = resultToDelimitedString(event, event.getResult(), saveConfiguration, delimiter);
+        out.println(save);
 
         if(saveConfiguration.saveSubresults()) {
             SampleResult result = event.getResult();
             for (SampleResult subResult : result.getSubResults()) {
-                savee = resultToDelimitedString(event, subResult, saveConfiguration, delimiter);
-                out.println(savee);
```

Comment:
I am not sure if `savee` meant the thing that is to be saved, so I will rename the variable to `formattedResult`. It seems to me, that it is exactly that.